### ISSUE TITLE
SKSE runtime 1.5.62

### ISSUE THREAD

**Issue by kotarf** [Sky-WAN/plugin#1]
Could you update the SKSE libraries with runtime 1.5.62? I've been trying to learn how this sample is configured in VS but having very little luck.

**Comment by flotwig** [comment]
I don't have a lot of time to work on this project right now, sorry. I will get to it when I have time. For now I would spend some time digging through the SKAAR repository which this is based off of or this repository: https://github.com/bedax/skse_plugin_example

Best of luck!

**Comment by kotarf** [comment]
Thank you very much for the advice. I'll start from the repo in your link.

**Comment by kotarf** [comment]
I ended up still using this sample project for the most part, and combing through its settings. I haven't worked with VS in about 6 years so you saved me a ton of time porting over my plugin to SKSE64. Thx!

**Comment by flotwig** [comment]
Ayy glad to have been of help! What are you working on, if I can ask?

**Comment by kotarf** [comment]
> Ayy glad to have been of help! What are you working on, if I can ask?

I made an SKSE-based patch for the original Locational Damage mod from 2012. It corrects several of its stability issues with respect to Papyrus and latency. I just ported this [patch ](https://www.nexusmods.com/skyrimspecialedition/mods/22443) to Special Edition & SKSE64 as of a few days ago. I needed to wait for PapyrusUtil to be updated also, but for a while I was struggling with configuring and compiling. That's where this sample delivered much needed aid =)
